lib/travis/build/script/r.rb

```diff
@@ -0,0 +1,368 @@
+module Travis
+  module Build
+    class Script
+      class R < Script
+        DEFAULTS = {
+          # Basic config options
+          cran: 'http://cran.rstudio.com',
+          warnings_are_errors: false,
+          # Dependencies (installed in this order)
+          apt_packages: [],
+          r_binary_packages: [],
+          r_packages: [],
+          bioc_packages: [],
+          r_github_packages: [],
+          # Build/test options
+          r_build_args: '--no-build-vignettes --no-manual',
+          r_check_args: '--no-build-vignettes --no-manual --as-cran',
+          r_check_cran_incoming: false,
+          r_check_revdep: false,
+          # Heavy dependencies
+          latex: false,
+          pandoc: false,
+          pandoc_version: '1.12.4.2',
+          # Bioconductor
+          bioc: 'http://bioconductor.org/biocLite.R',
+          bioc_required: false,
+          bioc_use_devel: true,
```

Comment:
you're totally right about the default: i just copied whatever was in r-travis.

i switched it, because we don't support `R-devel`, which is a requirement for bioc-devel. i do have plans in the works for making `R-devel` available, but it'll be a follow-on to this PR (since a few other pieces need to fall into place to make it easier).